pkg/scheduler/framework/plugins/volumebinding/volume_binding_test.go

```diff
@@ -878,6 +885,15 @@ func TestVolumeBinding(t *testing.T) {
 				assert.Equal(t, item.wantFilterStatus[i], gotStatus)
 			}
 
+			t.Logf("Verify: PreScore")
```

Comment:
Just make it consistent with other logs.

```suggestion
			t.Logf("Verify: call PreScore and check status")
```